### ISSUE TITLE
[BUG] v1.1.2-alpha Fails to Start

### ISSUE THREAD

**Issue by Ablex222** [Inpacchi/Manga-Tagger#22]
**OS**: What operating system are you running Manga Tagger on?
Windows 10
**Manga Tagger Version:** What version of Manga Tagger are you running?
v1.1.2-alpha
**Description of the issue:** Give a a clear and concise description of what the issue is.
Starting new version once selecting FMD folder proceeds to crash and give error.
**Manga Series:** What manga series (singular or plural) were being tagged when the issue was encountered?

**Screenshots (If applicable):** Add screenshots to help explain your problem.
![image](https://user-images.githubusercontent.com/77203222/104729002-c666a200-5705-11eb-9fc6-fdc699bf5fb3.png)

**Log File**: Please make sure JSON logging is enabled and upload the JSON log file.

**Comment by Inpacchi** [comment]
Ah, I overlooked a setting in the settings.json. This is a quick fix; will handle it later today.

**Comment by Ablex222** [comment]
Was wondering if this got resolved?

**Comment by Inpacchi** [comment]
Meant to respond to this yesterday but had to run out the door. I did start working on it, and was in the middle of QAing before I had to stop for the night, and then the weekend happened and you know how that goes.  

I picked up work on this last night (I actually was streaming my bug fixing) and made some good progress. It was originally going to be a quick fix however, when I QA’d (because I forgot to QA the start up process; I only QA’d the other changes I made) I noticed that FMD wasn’t opening or closing. I’m currently in the middle of debugging that and making sure FMD closes as it should before Manga Tagger continues to do anything; otherwise the application won’t work correctly.
